packages/react-component-library/src/fragments/Sidebar/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react'
+
+import { Avatar, Link } from '../../components'
+import NotificationSidePanel from '../NotificationSidePanel'
+
+interface UserLinkType {
+  initials: string
+}
+
+interface SidebarProps {
+  navItems: NavItemTypes[]
+  NotificationsPopoverContent?: JSX.Element
+  unreadNotification?: boolean
+  user?: UserLinkType
+  LinkComponent?: any
+}
+
+const Sidebar: React.FC<SidebarProps> = ({
+  LinkComponent = Link,
+  navItems,
+  NotificationsPopoverContent,
+  unreadNotification = false,
+  user,
+}) => {
+  const [open, setOpen] = useState()
+
+  const renderUserLink = ({ initials, ...rest }: UserLinkType): JSX.Element => (
+    <LinkComponent className="rn-sidebar__nav-link" {...rest}>
+      <Avatar
```

Comment:
Is `Avatar` independently documented and can it be used elsewhere or is it tightly coupled to the Sidebar at the moment?

Reply:
Makes sense to document it on it's own as it's a component. I'll do that in a follow up